### ISSUE TITLE
Add db alias and db description to quick search

### ISSUE THREAD

**Issue by fbemm** [TBroTeam/TBro#34]
Laura DeMare from Genome Research requested to include the alias (synonym) and the description of unigenes/isogorms to be indexed and searchable. Would be great if we can get this done. Would make things easier for people that search for their favorite gene. In parallel we should add a descriptive page to TBro that explains what annotation comes from where and how reliable it is.

**Comment by greatfireball** [comment]
Index means creating a database index (which might not be verified by any reviewer and has only impact on search speed) or creating of a special index page listing all alias and makes no sense for descriptions.

**Comment by greatfireball** [comment]
For the second issue "Description page" I created another issue #35 

Therefore this issue should only cover the search part of @fbemm request.

**Comment by iimog** [comment]
The db name (fullname) is already searched for in the quicksearch, the description can be included as well. It might slow down the search but I will try. As for the index I need further clarification.

**Comment by iimog** [comment]
I extended the search on the description field. 
![screenshot_desc](https://cloud.githubusercontent.com/assets/7403236/14518080/8752724c-0214-11e6-955e-77840bc9e878.png)
![screenshot_syn](https://cloud.githubusercontent.com/assets/7403236/14518079/874fb9a8-0214-11e6-9eae-7c6eab464250.png)

I further expanded the expression to allow matches that are not at the beginning of the string. Performance does not seem to be impacted too much (the LIMIT in the SQL statement prevents overly long execution times).

@fbemm please confirm that the issue is thereby resolved or specify what else has to be done.
As soon as I have your go I will apply the changes on the production machine (for now I have them only locally)

**Comment by fbemm** [comment]
Looks good! +1
